From 40a67469310c7f7fab1d1725d7c32f36c510468a Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Fri, 28 Jul 2017 12:38:28 +0800
Subject: [PATCH 2244/5242] MLK-16097-1 ARM64: imx8qxp: distinguish qm and qxp
 for pcie

commit  842863d3b59667b6ac301dd2f73adfb86b86d1fb from
https://source.codeaurora.org/external/imx/linux-imx.git

Use imx8qxp to specify the imx8qxp pcie support

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/pci/fsl,imx6q-pcie.txt     |    1 +
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi     |    2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/pci/fsl,imx6q-pcie.txt b/Documentation/devicetree/bindings/pci/fsl,imx6q-pcie.txt
index 91aa2c9..30c8e9f 100644
--- a/Documentation/devicetree/bindings/pci/fsl,imx6q-pcie.txt
+++ b/Documentation/devicetree/bindings/pci/fsl,imx6q-pcie.txt
@@ -10,6 +10,7 @@ Required properties:
 	- "fsl,imx6qp-pcie"
 	- "fsl,imx7d-pcie"
 	- "fsl,imx8qm-pcie"
+	- "fsl,imx8qxp-pcie"
 	- "fsl,imx8mq-pcie"
 - reg: base address and length of the PCIe controller
 - interrupts: A list of interrupt outputs of the controller. Must contain an
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index c9d29df..fd14e2d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -1812,7 +1812,7 @@
 		 * pcieb phyx1 lane1 in default, adjust it refer to the
 		 * exact hw design.
 		 */
-		compatible = "fsl,imx8qm-pcie","snps,dw-pcie";
+		compatible = "fsl,imx8qxp-pcie","snps,dw-pcie";
 		reg = <0x0 0x5f010000 0x0 0x10000>, /* Controller reg*/
 		      <0x0 0x7ff00000 0x0 0x10000>; /* PCI cfg space */
 		reg-names = "dbi", "config";
-- 
1.7.9.5

